cura/Arranging/ArrangeObjectsJob.py

```diff
@@ -39,10 +39,17 @@ def run(self):
 
         arranger = Arrange.create(x = machine_width, y = machine_depth, fixed_nodes = self._fixed_nodes, min_offset = self._min_offset)
 
+        # Build set to exclude children (those get arranged together with the parents).
+        included_as_child = set([])
```

Comment:
Why not just use set() (i don't really see the point of constructing a set with an empty list.)

Reply:
Leftover from before I decided that this should be a set probably. I'll change it.